### PR TITLE
feat(incidents): Add container component for incidents

### PR DESCRIPTION
**Pull request by lynnagara**
Add container for the incidents list and detail views behind the
organization:incidents feature flag.

### CODE REVIEW

File: src/sentry/static/sentry/app/routes.jsx [hunk 1]
```diff
@@ -908,6 +908,21 @@ function routes() {
               component={errorHandler(LazyLoad)}
             />
           </Route>
+          <Route
+            path="/organizations/:orgId/incidents/"
+            componentPromise={() =>
+              import(/* webpackChunkName: "OrganizationIncidentsContainer" */ './views/organizationIncidents/index')
```
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: What's the reason for explicitly including `index` here? Doesn't that defeat the purpose of an index file?

File: src/sentry/static/sentry/app/routes.jsx [hunk 2]
```diff
@@ -908,6 +908,21 @@ function routes() {
...
+            }
+            component={errorHandler(LazyLoad)}
+          >
+            <IndexRoute
+              componentPromise={() =>
+                import(/* webpackChunkName: "OrganizationIncidents" */ './views/organizationIncidents/list/index')
```
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: Should the chunk name be "OrganizationIncidentsList"?

File: src/sentry/static/sentry/app/views/organizationIncidents/index.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  renderNoAccess() {
+    return <Alert type="warning">{t("You don't have access to this feature")}</Alert>;
+  }
+
+  render() {
+    const {organization, children} = this.props;
+
+    return (
+      <DocumentTitle title={`Incidents - ${organization.slug} - Sentry`}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationIncidents/index.jsx]: Won't the children views be defining the doc title?

File: src/sentry/static/sentry/app/routes.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/routes.jsx]: Just seems to be done like this in a few places in the routes file, I can remove if you like

File: src/sentry/static/sentry/app/routes.jsx (hunk 2 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/routes.jsx]: Just copying the naming structure of monitors but I can update to call the chunk list

File: src/sentry/static/sentry/app/views/organizationIncidents/index.jsx (hunk 3 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationIncidents/index.jsx]: Removed

File: src/sentry/static/sentry/app/routes.jsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/routes.jsx]: Yeah can we remove all of the `/index` imports?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/routes.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/routes.jsx]: Sounds good to me

**Pull request merged**